databox/client/lib/DataboxInternal.ts

```diff
@@ -12,16 +24,24 @@ export default class DataboxInternal extends TypedEventEmitter<{ close: void; er
 
   #output: Output;
   #isClosing: Promise<void>;
+  #extractorPromises: Promise<any>[] = [];
 
   constructor(runOptions: IDataboxRunOptions) {
     super();
     this.runOptions = runOptions;
     this.initializeHero();
+    this.coreSessionPromise
+      .then(coreSession => databoxInternalByCoreSession.set(coreSession, this))
+      .catch(error => console.log('Could not set databoxInternalByCoreSession: ', error));
```

Comment:
Are we going to just use console.log in databox? FWIW, seems like this should be a console.error based on other console statements.

Reply:
Good question. Now that I think about it, there is no need for this console.log. Removed.